src/app/apps/page.tsx

```diff
@@ -1,24 +1,32 @@
-import { getApps } from "@/badgehub-api-client";
+import {getApps, getCategories, getDevices} from "@/badgehub-api-client";
```

Comment:
Looks like the style here changed from having spaces inside the braces to not - was that Prettier?

Reply:
I think I used the formatter of my IDE 🙂